src/App/Fossa/ManualDeps.hs

```diff
@@ -169,7 +169,7 @@ scanAndUpload root vdeps vendoredDepsOptions = do
   (archiveOrCLI, vendoredDependencyScanMode) <- getScanCfg org vendoredDepsOptions
   let scanner = case archiveOrCLI of
         ArchiveUpload -> archiveUploadSourceUnit
```

Comment:
We should emit a warning for archive upload or outright fatal failure - if license path filtering is used. 

My personal preference is that we fail outright (I think this is the best time to do this - since we will not be breaking anyone and will avoid future support tickets re this)